src/hooks/useHover.ts

```diff
@@ -0,0 +1,14 @@
+import {useState} from 'react';
+
+const useHover = () => {
```

Comment:
Hoverable component has the same problem as the PressableWIthFeedback - it exposes the `hovered` value via children as function. While `OfflineWithFeedback` uses `cloneElement` approach to add some additional props to its children, it can't add it to children that are not valid React elements (function in this case). So we need another way to check if the element is hovered without breaking `OfflineWithFeedback` functionality.